cmd/metrics-v2.go

```diff
@@ -1066,6 +1070,95 @@ func getILMNodeMetrics() MetricsGroup {
 	}
 }
 
+func getScannerNodeMetrics() MetricsGroup {
```

Comment:
isn't this a cluster metrics? @klauspost

Reply:
@harshavardhana It is recorded by scanners running locallly on disks, so it will have to be accumulated for cluster metrics.